### PR TITLE
Pp 5243 Remove One-Off Mandate from DDConnector

### PR DESCRIPTION
**Pull request by oswaldquek**

### CODE REVIEW

**Comment by oswaldquek** [comment]
@simad i've deleted all the things

File: src/main/java/uk/gov/pay/directdebit/mandate/services/MandateService.java [hunk 1]
```diff
@@ -126,10 +121,7 @@ public TokenExchangeDetails getMandateFor(String token) {
                 .map(mandateStateUpdateService::tokenExchangedFor)
                 .orElseThrow(TokenNotFoundException::new);
 
-        String transactionExternalId = mandate.getType().equals(MandateType.ONE_OFF)
-                ? retrieveTransactionForOneOffMandate(mandate.getExternalId()).getExternalId()
-                : null;
-        return new TokenExchangeDetails(mandate, transactionExternalId);
+        return new TokenExchangeDetails(mandate, null);
```
Comment (stephencdaly) [inline src/main/java/uk/gov/pay/directdebit/mandate/services/MandateService.java]: Can the transactionExternalId field be removed from TokenExchangeDetails?

File: src/main/java/uk/gov/pay/directdebit/tokens/api/TokenResponse.java [hunk 2]
```diff
@@ -33,9 +33,6 @@
     @JsonSerialize(using = ToStringSerializer.class)
     private MandateBankStatementReference mandateReference;
 
-    @JsonProperty("mandate_type")
-    private String mandateType;
-
     @JsonProperty("transaction_external_id")
     private String transactionExternalId;
```
Comment (stephencdaly) [inline src/main/java/uk/gov/pay/directdebit/tokens/api/TokenResponse.java]: Can we remove this field?

**Comment by simad** [comment]
Despite a few ..slips here and there (cough @alexbishop1) removing code makes me really happy 💃 @oswaldquek delete all the things!

File: src/main/java/uk/gov/pay/directdebit/tokens/api/TokenResponse.java (hunk 2 above)
Comment (oswaldquek) [reply src/main/java/uk/gov/pay/directdebit/tokens/api/TokenResponse.java]: done

File: src/main/java/uk/gov/pay/directdebit/mandate/services/MandateService.java (hunk 1 above)
Comment (oswaldquek) [reply src/main/java/uk/gov/pay/directdebit/mandate/services/MandateService.java]: done

**Pull request merged**